src/views/DynamicRulesList.vue

```diff
@@ -0,0 +1,259 @@
+<template>
+  <div class="card-content">
+    <div class="media">
+      <div class="media-content">
+        <div class="field is-grouped">
+          <p class="control">
+            <button class="button is-small download-doc-button"
+                    :class="{'is-loading':isDownloadLoading}"
+                    @click="downloadDoc()"
+                    title="Download document"
+                    data-qa="download-document">
+                <span class="icon is-small">
+                    <i class="fas fa-download"></i>
+                </span>
+            </button>
```

Comment:
Changed from `main` branch:

```suggestion
        <div class="field is-grouped is-pulled-right">
          <p class="control">
            <button class="button is-small download-doc-button"
                    :class="{'is-loading':isDownloadLoading}"
                    @click="downloadDoc()"
                    title="Download document"
                    data-qa="download-document">
              <span class="icon is-small">
                <i class="fas fa-download"></i>
              </span>
              <span>
                Download
              </span>
            </button>
```